### ISSUE TITLE
Cannot save photos in Android app (locally created files only)

### ISSUE THREAD

**Issue by kevin-garcia** [lutraconsulting/input#546]
I'm experiencing a strange behavior in the Android app that only seems to happen with locally created files (files in public projects that I download behave fine). When I try to add a photo to a record in Input, I get the camera icon, but after I press on it, I don't get the check/x to save or discard the image:

![image](https://user-images.githubusercontent.com/16250189/70015343-f6d89480-1574-11ea-8022-380bd7c362f5.png)
![image](https://user-images.githubusercontent.com/16250189/70015430-356e4f00-1575-11ea-95da-faa07261ca7a.png)

When I try this with public projects downloaded to my device, it works fine:

![image](https://user-images.githubusercontent.com/16250189/70015509-83835280-1575-11ea-9202-d923d9169f7f.png)
![image](https://user-images.githubusercontent.com/16250189/70015723-1b813c00-1576-11ea-9810-50878a5c67c3.png)

I've been careful to ensure that the field settings are exactly the same for my locally created files as for the ones I've downloaded, and I've tested and retested multiple files, all with the same result: the locally created files just don't give me the option to save or discard, while files from the public projects I download do.

**Comment by saberraz** [comment]
Could you share a screenshot of your photo edit widget in QGIS? It should look something like this:
![ftogXdLr](https://user-images.githubusercontent.com/2649047/70040013-b5ff7080-15b2-11ea-95a8-2e4a085f2d3d.png)

**Comment by kevin-garcia** [comment]
I've tried all of the combinations for the photo edit widget I can think of, and none work. Here's the current setting I have:

![image](https://user-images.githubusercontent.com/16250189/70156435-d5c68f80-16ab-11ea-845f-50349174788d.png)

I've also set the widget settings to match the settings from project files that I've downloaded (and work when I load them into Input), and I get the same result: both files have the same settings, but the locally created one doesn't work, while the downloaded one does.

I've been using QGIS 3.10. I'm going to try creating a project with QGIS 3.4 to see if that makes a difference, as I assume a lot of the public projects were made prior to 3.10 and 3.4 is the LTR.

**Comment by saberraz** [comment]
Hi @kevin-garcia 

Could you share your project with me on mergin (my username is saber)

Thanks

**Comment by saberraz** [comment]
I have just tried with QGIS master and it works fine.

**Comment by kevin-garcia** [comment]
Hi Saber,

Thanks so much for your email. You solved my problem, which happened because I had not set a CRS for my project. I'll repeat some of your email below to help others who may have this same issue. (One thing I noticed is that this only was an issue when I made a project in QGIS 3.10. I tried a test project in 3.4 and the photo capture worked fine, even without setting a project CRS.)

_To set the project CRS, go to QGIS Desktop > Project > Properties and a new window opens.
From the left hand side, select CRS tab and search for 3857 (Pseudo Mercator)._
 
![image](https://user-images.githubusercontent.com/16250189/70170703-1d0d4a00-16c5-11ea-8578-6ae3a48c28e6.png)
 
_Press OK and save your project. Then upload to Mergin and download it through the app. It should work after that._

Thanks again!
-Kevin
